### PR TITLE
Add com.snowplowanalytics.snowplow/web_page/jsonschema/1-1-0 (closes #1222)

### LINKED ISSUE #1222: Add com.snowplowanalytics.snowplow/web_page/jsonschema/1-1-0
Add the `tabId` property on the web_page schema. Based on https://github.com/snowplow/snowplow-javascript-tracker/pull/1088

### PR DESCRIPTION
**Pull request by igneel64**
Closes #1222 

- Both `id` and `tabId` are generated using the [uuid](https://www.npmjs.com/package/uuid) pacakge which produces valid UUIDs. 
- `tabId` is the identifier of the current tab the event was sent from. More info on the [JS repository](https://github.com/snowplow/snowplow-javascript-tracker/pull/1088).

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: schemas/com.snowplowanalytics.snowplow/web_page/jsonschema/1-1-0 [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "web_page",
+        "format": "jsonschema",
+        "version": "1-1-0"
+    },
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+			"format": "uuid",
```
Comment (matus-tomlein) [inline schemas/com.snowplowanalytics.snowplow/web_page/jsonschema/1-1-0:14]: This seems like a combination of tabs and spaces.

**Review by matus-tomlein — COMMENTED** [review]
Looks good!

We do have slightly different conventions when it comes to contributing to Iglu central. Here is [a guideline specific to contributing to Iglu central](https://github.com/snowplow-incubator/engineering-resources-internal/blob/master/iglu-central/workflow.md). Just a couple of nitpicks:
* The title of both the PR and the issue should be `Add SCHEMA_URL`
* The branch name should contain underscores in the description part, so `feature/1222-add_tabid_to_webpage`

File: schemas/com.snowplowanalytics.snowplow/web_page/jsonschema/1-1-0 (hunk 1 above)
Comment (igneel64) [reply schemas/com.snowplowanalytics.snowplow/web_page/jsonschema/1-1-0:14]: Yeap fixing that.

**Pull request closed without merging**